STAT302 - Assignment 2 Q 2.26

In [1]:
#Q 2.26 a
library(Stat2Data)
data(LeafWidth)
str(LeafWidth)

'data.frame':	252 obs. of  5 variables:
 $ Width  : num  0.658 0.921 0.921 0.921 0.921 ...
 $ Length : num  85.1 53.2 59.9 63.7 64.2 ...
 $ LWRatio: num  129.4 57.7 65 69.1 69.7 ...
 $ Area   : num  70.3 49 57.7 37.4 62.6 ...
 $ Year   : int  1974 1918 1993 1946 1956 1964 1956 1980 1999 1977 ...


In [2]:
cor.test(LeafWidth$Width, LeafWidth$Year)


	Pearson's product-moment correlation

data:  LeafWidth$Width and LeafWidth$Year
t = -4.0294, df = 250, p-value = 7.425e-05
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.3595491 -0.1272589
sample estimates:
       cor 
-0.2469483 


H0: \rho = 0, Ha: \rho /= 0 \
t = -4.0294, p-value = 7.425e-05 \
reject H0 \
We have evidence to prove width and year have linear relationship.

In [3]:
#Q 2.26 b
R2 = (-0.2469)^2
R2

[1] 0.06095961

In [4]:
#Q 2.26 c
model = lm(Width ~ Year, data = LeafWidth)
anova(model) # summary(aov(model))

Df  Sum Sq   Mean Sq   F value Pr(>F)      
Year        1  32.9113 32.911298 16.236  7.425211e-05
Residuals 250 506.7643  2.027057     NA            NA

F = 16.24, p-value = 7.43e-05 \
reject H0 \
We have evidence that Width and Year have linear relationship.

In [5]:
#Q 2.26 d
sqrt(16.24)

[1] 4.029888

So: ANOVA's F should be equal to t^2